lib/ansible/modules/cloud/amazon/iam_cert.py

```diff
@@ -269,10 +281,23 @@ def main():
     cert_chain = module.params.get('cert_chain')
     dup_ok = module.params.get('dup_ok')
     if state == 'present':
-        cert = open(module.params.get('cert'), 'r').read().rstrip()
```

Comment:
I'll rebase this up to current devel again. Unfortunately since I'm not actively using AWS at the moment, chasing these PRs has been a challenge.